### DISCUSSION TITLE
Eigen support

### DISCUSSION THREAD

**Discussion by johnwason** [carbon-language/carbon-lang#1723] [Language design]
Eigen linear algebra template library is very important for scientific and engineering applications. I suggest "But does it run Eigen?" as a test of interoperability between Carbon and C++ templates because of its importance, and also because it is a good example of a complex template library.

**Comment by jonmeow** [comment]
Honestly I think Eigen is a popular enough library that we'll run into it while trying to seriously test interop. But it's pretty far off right now that we'll be doing that.
